### PR TITLE
longevity: add counter_write workload in prepare_write stage

### PR DESCRIPTION
**Pull request by amoskong**
In some longevity jobs, we run counter_read & counter_write after prepare_write
stage, then counter_read workloads always fail for key missing.

This patch added counter_write workload in prepare_write stage if the job has
counter_read workload after prepare_write.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (`hydra unit-tests`)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by amoskong** [comment]
@bentsi @roydahan This PR is simple, it just add a counter_write in prepare_write stage.
Currently we run counter_read in 3 jobs without prepare data by counter_write.

File: test-cases/longevity/longevity-50GB-4days-authorization-and-tls-ssl.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 test_duration: 6550
-prepare_write_cmd: "cassandra-stress write cl=QUORUM n=100000000 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native  -rate threads=500 -pop seq=1..100000000"
+prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=100000000 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native  -rate threads=500 -pop seq=1..100000000",
+                    "cassandra-stress counter_write cl=QUORUM n=1000000 -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1 -pop seq=1..1000000"]
 stress_cmd: ["cassandra-stress mixed cl=QUORUM duration=5760m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native  -rate threads=20 -pop seq=1..100000000 -log interval=5",
              "cassandra-stress counter_write cl=QUORUM duration=5760m -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1 -pop seq=1..1000000",
```
Comment (bentsi) [inline test-cases/longevity/longevity-50GB-4days-authorization-and-tls-ssl.yaml]: why do we need to run `counter_write` c-s again?

**Comment by bentsi** [comment]
@roydahan in some of the commands we run `counter_write` c-s again, is it intended?

**Comment by roydahan** [comment]
I think it doesn't hurt, so we have some writes also during the main stress.
However, I'm not sure if it works or just stuck after the first round. @amoskong ?

**Comment by amoskong** [comment]
It's necessary to have both counter read and write in running nemesis.

Bentsi <notifications@github.com> 于 2019年7月24日周三 上午12:32写道：

**Comment by amoskong** [comment]
The first is used to prepare data, second is used to verify counter writing
works well during nemesis.

Bentsi <notifications@github.com> 于 2019年7月24日周三 上午12:34写道：

**Pull request merged**